Content.Server/GameObjects/Components/Sound/EmitSoundOnLandComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Shared.Interfaces.GameObjects.Components;
+using Robust.Shared.Audio;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Player;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Sound
+{
+    /// <summary>
+    /// Simple sound emitter that emits sound on land
+    /// </summary>
+    [RegisterComponent]
+    public class EmitSoundOnLandComponent : Component, ILand
```

Comment:
extracted this, as per the Issue's description. Perhaps we could also use an EmitSoundOnActivateComponent?

Reply:
thats what EmitSoundOnUse is